code/modules/clothing/shoes/magboots.dm

```diff
@@ -180,8 +180,12 @@
 	else
 		if(H.w_uniform.type == /obj/item/clothing/under/russobluecamooutfit || istype(H.w_uniform, /obj/item/clothing/under/neorussian))
 			russian+=2
-	if(findtext("ivan",lowertext(H.name)) || findtext("yuri",lowertext(H.name)) || findtext("vlad",lowertext(H.name)) || findtext("lenin",lowertext(H.name)) || findtext("boris",lowertext(H.name)) || findtext("sasha",lowertext(H.name)) || findtext("misha",lowertext(H.name)) || findtext("sergei",lowertext(H.name)))
+	if((findtext("ivan",lowertext(H.name)) || findtext("yuri",lowertext(H.name)) || findtext("vlad",lowertext(H.name) && !findtext("putin",lowertext(H.name))) || findtext("lenin",lowertext(H.name)) || findtext("boris",lowertext(H.name)) || findtext("sasha",lowertext(H.name)) || findtext("misha",lowertext(H.name)) || findtext("sergei",lowertext(H.name))) && !findtext("putin",lowertext(H.name)))
```

Comment:
Jesus this is a clusterfuck
Just swap the order of this check and the Putin check and make this one `else if()`

Reply:
That was the original plan but I thought maybe it could conflict with all the other ifs if I slap an else if and the russian value would be much lower

Reply:
It only applies to the previous `if()`

Reply:
Someone made a point yesterday while making this PR that if somewhere were to name themselves Vladimir Putin it would catch the Vlad in their name

Reply:
Not if you did what I suggested